supportedBackends/google/batch/src/main/scala/cromwell/backend/google/batch/runnable/RunnableBuilder.scala

```diff
@@ -160,11 +161,17 @@ object RunnableBuilder {
           .addCommands(scriptContainerPath)
           .setUsername(username)
           .setPassword(password)
+          .setOptions(
+            s"--shm-size=${toMemMib(memory) * 0.8}m"
+          ) // configure file system to have shared memory 80% of the memory size
       case _ =>
         Container.newBuilder
           .setImageUri(docker)
           .setEntrypoint(jobShell)
           .addCommands(scriptContainerPath)
+          .setOptions(
```

Comment:
There is an opportunity to avoid code duplication by doing something like this:

```scala
val containerBuilder = (dockerhubCredentials._1, dockerhubCredentials._2) match {
  case (username, password) if username.nonEmpty && password.nonEmpty =>
    Container.newBuilder.setUsername(username).setPassword(password)
  case _ =>
    Container.newBuilder
}
val container = containerBuilder.setImageUri(docker).... // etc
```